daemon/pod.go

```diff
@@ -42,11 +43,201 @@ type Pod struct {
 	sync.RWMutex
 }
 
-func NewPod(rawSpec []byte, id string, data interface{}) (*Pod, error) {
+// TODO: remove convertToRunvPodSpec after pod.UserPod is deleted from runv
+func convertToRunvPodSpec(podSpec *apitypes.UserPod) (*pod.UserPod, error) {
```

Comment:
pod.UserPod is already user interface, it will not be removed(it will be moved back to hyperd)

convertToRunvPodSpec should be convertToUserPod for respecting the name.
I suggest to change apitypes.UserPod to apitypes.Pod